config/codecov.map

```diff
@@ -16,7 +16,7 @@
 ^/(.*)/(.*)/(.*)/branch/(?P<branch>.+)/(graph|graphs)/badge.svg be_api
 ^/.*/.*/.*/(graph|graphs)/badge.svg be_api
 ^/login/(gh|github|ghe|github_enterprise|bb|bitbucket|bbs|bitbucket_server|gl|gitlab|gle|gitlab_enterprise|okta|sentry) be_api
-^/logout/(gh|github|ghe|github_enterprise|bb|bitbucket|bbs|bitbucket_server|gl|gitlab|gle|gitlab_enterprise|okta|sentry) be_api
+^/logout((\/)(gh|github|ghe|github_enterprise|bb|bitbucket|bbs|bitbucket_server|gl|gitlab|gle|gitlab_enterprise|okta|sentry)?)?$
```

Comment:
Is there a reason `be_api` is missing?

Reply:
I think you could get away with

```
^/logout\/?(gh|github|ghe|github_enterprise|bb|bitbucket|bbs|bitbucket_server|gl|gitlab|gle|gitlab_enterprise|okta|sentry)?$
``` to be cleaner

Reply:
good catch! 😲

Reply:
I also am not sure if you need the `$` at the end, perhaps if the url ends with a trailing `/`

Reply:
I think you're right